giftcard.adoc

```diff
@@ -41,6 +41,6 @@ uuid:: The UUID of the giftcard
 The `giftcards` data type only contain a few generic parameters such as `uuid`, `amount` and `dates`.
 
 - `created` the date that the giftcard was created
-- `validTo` once this date has passed the giftcard will be expired and can no longer be used
+- `validTo` once this date has passed the giftcard will be expired and can no longer be used. If the value is null, giftcard will never expire.
```

Comment:
should we add something about that it can only be null if it's allowed to never expiry in the market that the seller operates in or something or is that to much information here?

Reply:
Yes, I felt that it's too much information to be mentioned in the API.